examples/dual_variable_example/dual_variable_example.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+
+"""
+General description
+-------------------
+
+A basic example to show how to get the dual variables from the system. Try
+to understand the plot.
```

Comment:
It would be nice to have an explanation for the plot. At the moment, it's really hard to understand.

Reply:
For me it is not to complicated. It is always good to have more docs but I think it is pretty self explaining. But for sure you have to understand how dual variables work, but this is not our task.